tsdb/compact.go

```diff
@@ -559,6 +559,7 @@ func (c *LeveledCompactor) CompactWithBlockPopulator(dest string, dirs []string,
 }
 
 func (c *LeveledCompactor) Write(dest string, b BlockReader, mint, maxt int64, base *BlockMeta) ([]ulid.ULID, error) {
+	c.logger.Info("write block started")
```

Comment:
What about adding mint, and maxt (maybe also ULID) details while we have those available cheaply?

Reply:
added